config/kernelci.conf

```diff
@@ -28,6 +28,8 @@ lab_config: shell
 email_host: your.host
 email_port: your.port
 
+[trigger_completed]
```

Comment:
@JenySadadia which kind of  `conf ` could we need to `trigger_completed`?

Reply:
Atm, we don't have any configuration needed for the service. I have added it to the file for future purposes just like we have sections for `kcidb` and `notifier`.

Reply:
There's at least `db_config`:
```python
class cmd_run(Command):
    help = "Trigger an event if all child nodes are completed"
    args = [Args.db_config]
```

Reply:
Yes, I meant we don't need any additional configurations except the ones specified in `DEFAULT`.